src/textual/devtools/renderables.py

```diff
@@ -21,10 +24,31 @@
 from rich.segment import Segment, Segments
 from rich.table import Table
 
-DevtoolsMessageLevel = Literal["info", "warning", "error"]
+DevConsoleMessageLevel = Literal["info", "warning", "error"]
+
+
+class DevConsoleHeader:
+    def __rich_console__(
+        self, console: Console, options: ConsoleOptions
+    ) -> RenderResult:
+        lines = Renderables(
+            [
+                f"[bold]Textual Development Console [#b169dd]v{textual.__version__}",
+                "[#967fa3]Run a Textual app with the environment variable [b]TEXTUAL_DEVTOOLS=1[/] to connect.",
```

Comment:
We probably don't want to use RGB colors here, for weirdos on a light color scheme or who don't want to deviate from their ANSI theme.

Reply:
I did initially try using ANSI colours here (and in the devtools log headers where we also use RGB), but honestly, I couldn't get it to look anywhere near as good.

I checked these colours look good on light themes too, and chose them to match the default purple Textual theme/textualize colours. My thinking was if you're building a Textual app we're already completely hijacking your colour scheme there anyway, so some subtle branding colours in the devtools should be no big deal?

<img width="657" alt="image" src="https://user-images.githubusercontent.com/5740731/164005420-e909afe6-e50a-4812-bc65-af8df8e8c85f.png">

Reply:
Makes sense, but I'm sure we'll get some complaints. Let's for for ANSI here.